userinterface/App.axaml.cs

```diff
@@ -123,4 +193,36 @@ protected static Bootstrapper BootstrapBackEnd()
             },
         };
     }
+
+    private async Task ShowAlphaBuildWarningAsync()
+    {
+        var modalService = Services?.GetService<IModalService>();
+        if (modalService != null)
+        {
+            var message = "This is an alpha build of RawAccel. You may encounter bugs or unexpected behavior.\n\n" +
+                          "You can report issues at: https://github.com/RawAccelOfficial/rawaccel/issues\n\n" +
+                          "There's also a bug report button in the Settings page for easy access.";
+
+            await modalService.ShowMessageAsync(
+                "Alpha Build Warning",
+                message,
+                "I Understand");
+        }
+    }
+
+    public static void OpenBugReportUrl()
```

Comment:
I like the messaging that this is an alpha build. Can we plan to remove the Bug Report button (or better hide it and couch with language indicating to search through existing issues etc.) once 2.0 is out of alpha? For this PR can you add comment indicating that?
If we have prominent reference to github issues, I fear deluge of issues that may not be "worthy".